Lib/pyclbr.py

```diff
@@ -45,33 +49,54 @@
 import tokenize
 from token import NAME, DEDENT, OP
 
-__all__ = ["readmodule", "readmodule_ex", "Class", "Function"]
+__all__ = ["readmodule", "readmodule_ex", "Object", "Class", "Function"]
 
 _modules = {}                           # cache of modules we've seen
 
-# each Python class is represented by an instance of this class
-class Class:
-    '''Class to represent a Python class.'''
-    def __init__(self, module, name, super, file, lineno):
+
+class Object:
+    """Class to represent a Python object."""
+    def __init__(self, module, name, file, lineno, parent):
         self.module = module
         self.name = name
+        self.file = file
+        self.lineno = lineno
+        self.parent = parent
+        self.objects = {}
+
+    def _addobject(self, name, obj):
+        self.objects[name] = obj
+
+
+# each Python class is represented by an instance of this class
```

Comment:
I edited docstrings and comments in the whole file and caught any fixes not done.